src/cryptoadvance/specter/api/API.md

```diff
@@ -0,0 +1,198 @@
+## Specter API
```

Comment:
Looks like this file is duplicated in two locations:
`src/cryptoadvance/specter/api/API.md` and `docs/API.md`.
Do we need both?